microsoft/testsuites/performance/storageperf.py

```diff
@@ -455,13 +464,34 @@ def _perf_premium_datadisks(
         max_iodepth: int = 256,
     ) -> None:
         disk = node.features[Disk]
-        data_disks = disk.get_raw_data_disks()
-        disk_count = len(data_disks)
-        assert_that(disk_count).described_as(
-            "At least 1 data disk for fio testing."
-        ).is_greater_than(0)
-        partition_disks = reset_partitions(node, data_disks)
-        filename = ":".join(partition_disks)
+        if disk.disk_controller_type == 'nvme':
```

Comment:
This does not match formatting guidelines. Please ensure your development environment is properly set up to automatically apply flake8 changes.

https://mslisa.readthedocs.io/en/main/write_test/dev_setup.html